torch/csrc/jit/passes/onnx/unpack_quantized_weights.cpp

```diff
@@ -40,7 +40,10 @@ double getScaleFromInput(Node* input_node) {
                                                  "aten::slice",
                                                  "aten::avg_pool2d",
                                                  "quantized::cat",
-                                                 "prim::ListConstruct"};
+                                                 "prim::ListConstruct",
```

Comment:
How do we test that the onnx conversion works correctly for the new ops?

Reply:
We already had tests, these paths get exercised only when we have a sequence of ops like upsample followed by conv. Conv needs scale argument so it gets it from the input tensor of these ops.